README.md

```diff
@@ -100,7 +100,7 @@ const csvFromArrayOfObjects = convertArrayToCSV(dataObjects);
   const csvFromArrayOfArrays  = 'number;first;last;handle\n1;Mark;Otto;@mdo\n2;Jacob;Thornton;@fat\n3;Larry;the Bird;@twitter\n';
 */
 const csvFromArrayOfArrays = convertArrayToCSV(dataArrays, {
-  header,
+  header: header,
```

Comment:
On L67 in this file we actually declare the header `const header = ['number', 'first', 'last', 'handle'];` so in this case the example would work just fine.

See the previous comments of @Qiming-Liu for more information.